tests/acl/test_acl.py

```diff
@@ -9,6 +9,7 @@
 import ptf.testutils as testutils
 import ptf.mask as mask
 import ptf.packet as packet
+import paramiko
```

Comment:
has this been used in other places ? looks new to me.

Reply:
Hi Abhishek @abdosi It has been used in ssh scripts. The reason I am using ssh instead of duthost.shell is because duthost shell is an ansible procedure and it prints the awk with grep "LA regexp" in the syslog which interferes with the Loganalyzer check. This is more of a clean way to ssh into the box and not interfere with the syslog. I have run the test multiple times on T2 profile and not seen any issues till now.

Reply:
Used the SSHClient from utilites file